hieradata/vagrant/common.yaml

```diff
@@ -226,6 +226,23 @@ profile::jenkinscontroller::jcasc:
             architecture: "amd64"
             labels:
               - docker-windows
+              - docker-windows-2019
+            useAsMuchAsPossible: true
+            spot: true
```

Comment:
```suggestion
```

Reply:
Preparing the PR to remove it from the other Windows agents, no need to include it in this one.

Reply:
See https://github.com/jenkins-infra/jenkins-infra/pull/2585

Reply:
The other PR is not merged. There is no reason to link the 2 of them.

Reply:
Yes sure, I'm just asking to not include spot: true here as this one isn't merged yet too.

Reply:
I don't mind doing a little bit more work on my other PR after this one is merged, it just seemed to me easier like that.